docs/reference/pages/panels.md

```diff
@@ -346,7 +346,7 @@ class BookPage(Page):
     # ...
 
     content_panels = Page.content_panels + [
-        InlinePanel('related_links', label="Related Links"),
+        InlinePanel('related_links', heading="Related Links", label="Related link"),
```

Comment:
Here and below – `heading` is still optional and defaulting to `label`, but now that `label` is used as the label for each item as well as the Add button, I think we should encourage people to set both separately.

Having the heading be the plural form of the label is the simplest, but it could also be that a shorter label ("Link") works just as well.

---

I think we should mention this in the Upgrade considerations for the release.